styles/map.css

```diff
@@ -18,201 +20,253 @@ html, body, #map {
 
 ::-webkit-scrollbar-track {
```

Comment:
Wieso werden hier WebKit-spezifische Sachen gesetzt? Auch in CSS gibt es Kommentare!